eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -335,19 +344,26 @@ protected boolean isHistoryEnabled() {
     protected void run() {
         try {
             long start = System.currentTimeMillis();
-            logger.debug("Starting poll...");
+            logger.debug("Starting historical poll...");
             executeSync();
-            logger.debug("Poll completed in {}", FormatUtil.formatDurationSince(start));
+            logger.debug("Historical poll completed in {}", FormatUtil.formatDurationSince(start));
         } catch (Exception e) {
-            logger.error("Error polling Ewon data.", e);
+            logger.error("Error completing historical poll.", e);
         }
     }
 
     /**
      * Perform a synchronization of realtime tags
      */
     protected void runLive() {
-        updateLive();
+        try {
```

Comment:
This chunk of code should probably only be included if there is an option to set advanced mode, is there any real value to having this statistic? Right now it is really spitting out a lot of output to my log.

Reply:
Added this since its logged in run(), so I think it should be logged in runLive() too. I can add the option for verbose logging to this PR if you want